### PR TITLE
5138: Adjusted the chunk size for when we update the tables along wit…

### PR DESCRIPTION
**Pull request by johnathan-rodriguez**
…h some code cleanup.

### CODE REVIEW

File: amazonredshift_blueprints/upload_file.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
-from sqlalchemy import create_engine
```
Comment (johnathan-rodriguez) [inline amazonredshift_blueprints/upload_file.py:1]: my ide reorganizes the imports and removes unused modules

File: amazonredshift_blueprints/upload_file.py [hunk 2]
```diff
@@ -76,21 +76,10 @@ def get_args():
     return args
 
 
-def create_connection_url(host:str, password:str, user:str, database:str,port = 5439):
-    url = URL.create(drivername= 'redshift+redshift_connector', host = host, password= password, username= user, port = port, database=database)
+def create_connection_url(host: str, password: str, user: str, database: str, port=5439):
+    url = URL.create(drivername='redshift+redshift_connector', host=host, password=password, username=user, port=port,
+                     database=database)
     return url
-def create_connection_string(args):
```
Comment (johnathan-rodriguez) [inline amazonredshift_blueprints/upload_file.py:82]: I couldn't see anywhere where this was being used. I imagine this was used at one point but wasn't cleaned up

File: amazonredshift_blueprints/upload_file.py [hunk 3]
```diff
@@ -147,76 +137,46 @@ def map_datatypes(pandas_type:str) -> str:
     else:
         return 'VARCHAR(255)'
 
-def create_table_query(table_name:str, df:pd.DataFrame, database:str, schema:str = 'public') -> str:
```
Comment (johnathan-rodriguez) [inline amazonredshift_blueprints/upload_file.py:150]: being handled via the pandas module

File: amazonredshift_blueprints/upload_file.py [hunk 4]
```diff
@@ -235,12 +196,16 @@ def main():
     if schema == '':
         schema = None
 
-    db_string = create_connection_url(host = host, password = password, user = user, database = database)
+    if port in ('', None):
+        db_string = create_connection_url(host=host, password=password, user=user, database=database)
```
Comment (johnathan-rodriguez) [inline amazonredshift_blueprints/upload_file.py:177]: Decided to go this route so that the default value for the create connection function could be use if port comes in blank. Port wasn't being used in the script prior

File: amazonredshift_blueprints/upload_file.py [hunk 5]
```diff
@@ -251,14 +216,12 @@ def main():
         for index, key_name in enumerate(matching_file_names):
```
Comment (johnathan-rodriguez) [inline amazonredshift_blueprints/upload_file.py]: Not sure if this would work when using insert-method = replace. We should test and if it can't be used with replace we should handle the error

File: amazonredshift_blueprints/upload_file.py [hunk 6]
```diff
@@ -147,73 +137,44 @@ def map_datatypes(pandas_type:str) -> str:
...
-    else:
-        for index, chunk in enumerate(
-                pd.read_csv(source_full_path, chunksize=10000)):
-
-            if insert_method == 'replace' and index > 0:
-                # First chunk replaces the table, the following chunks
-                # append to the end.
-                insert_method = 'append'
-
+                chunksize=1000
```
Comment (wrp801) [inline amazonredshift_blueprints/upload_file.py]: Let's go back to the 10k chunk size, I don't see a valid reason to switch

File: amazonredshift_blueprints/upload_file.py (hunk 2 above)
Comment (wrp801) [reply amazonredshift_blueprints/upload_file.py:82]: Yep this was a legacy function, fine to remove

File: amazonredshift_blueprints/upload_file.py [hunk 7]
```diff
@@ -125,7 +114,8 @@ def combine_folder_and_file_name(folder_name, file_name):
 
     return combined_name
 
-def map_datatypes(pandas_type:str) -> str:
+
+def map_datatypes(pandas_type: str) -> str:
```
Comment (wrp801) [inline amazonredshift_blueprints/upload_file.py]: Since you're getting rid of the `create_table_query` function, you should remove this too since that is the only place where it is used

File: amazonredshift_blueprints/upload_file.py (hunk 5 above)
Comment (wrp801) [reply amazonredshift_blueprints/upload_file.py]: Going to test locally to see if this works

File: amazonredshift_blueprints/upload_file.py [hunk 8]
```diff
@@ -251,14 +216,12 @@ def main():
         for index, key_name in enumerate(matching_file_names):
             upload_data(
                 source_full_path=key_name,
-                table_name=table_name,
```
Comment (wrp801) [inline amazonredshift_blueprints/upload_file.py:254]: Why did you remove the `table_name` parameter?

**Review by wrp801 — CHANGES_REQUESTED** [review]
Small changes to make, I'm going to test the regex match with replace to see if that functions as expected

File: amazonredshift_blueprints/upload_file.py (hunk 8 above)
Comment (johnathan-rodriguez) [reply amazonredshift_blueprints/upload_file.py:254]: good catch- i believe this was a selection error when removing the database param

**Review by wrp801 — APPROVED** [review]
